src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/Cells/__tests__/HealthCertificateQRCodeCellViewModelTests.swift

```diff
@@ -2320,6 +2379,7 @@ class HealthCertificateQRCodeCellViewModelTests: XCTestCase {
 		XCTAssertEqual(viewModel.title, "Genesenenzertifikat")
 		XCTAssertNil(viewModel.subtitle)
 		XCTAssertEqual(viewModel.qrCodeViewModel.accessibilityLabel, "accessibilityText")
+		XCTAssertEqual(viewModel.qrCodeViewModel.covPassCheckInfoPosition, .top)
```

Comment:
Just wondering where the other positions are tested.

Reply:
For the `HealthCertificateQRCodeCellViewModel` it's actually always `.top` so all the tests check for it, but in the `HealthCertifiedPersonCellModelTests` you can see that it's mostly `.bottom` there.